runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -223,6 +220,7 @@ public void render(Graphics2D graphics)
 
 				if (overlay.getPosition().equals(OverlayPosition.DYNAMIC))
 				{
+					OverlayUtil.setGraphicProperties(graphics);
```

Comment:
Why does this need to be invoked for every overlay?

Reply:
It doesn't. I will put it back up there. My thinking was that it does not need to be invoked for non-dymanic overlay, but there is a loop -_-